model-optimizer/extensions/ops/non_max_suppression.py

```diff
@@ -46,17 +46,14 @@ def __init__(self, graph: Graph, attrs: dict):
         super().__init__(graph, mandatory_props, attrs)
 
     def supported_attrs(self):
```

Comment:
nit: why is it supported attrs? not backend attrs? seems we do not have that much of a difference between them.

Reply:
Changed